addons/account_audit_trail/models/mail_message.py

```diff
@@ -0,0 +1,82 @@
+        string="Journal Entry",
+        compute="_compute_account_audit_log_move_id",
+        search="_search_account_audit_log_move_id",
+    )
+    show_audit_log = fields.Boolean(compute="_compute_account_audit_log_move_id", search="_search_show_audit_log")
+
+    @api.depends('body', 'subject', 'tracking_value_ids', 'subtype_id')
+    def _compute_account_audit_log_preview(self):
+        for message in self:
+            title = message.subject or message.preview
+            tracking_value_ids = message.sudo().tracking_value_ids
+            if not title and tracking_value_ids:
+                title = _("Updated")
+            elif not title and message.subtype_id and not message.subtype_id.internal:
+                title = message.subtype_id.display_name
+            audit_log_preview = Markup("<div>%s</div>") % (title)
+            trackings = [
+                (
+                    fmt_vals['changedField'],
+                    fmt_vals['oldValue']['value'],
+                    fmt_vals['newValue']['value'],
+                ) for fmt_vals in tracking_value_ids._tracking_value_format()
+            ]
+            for field_desc, old_value, new_value in trackings:
+                audit_log_preview += Markup(
+                    "<li>%(old_value)s <i class='o_TrackingValue_separator fa fa-long-arrow-right mx-1 text-600' title='%(title)s' role='img' aria-label='%(title)s'></i>%(new_value)s (%(field)s)</li>"
+                ) % {
+                    'old_value': old_value,
+                    'new_value': new_value,
+                    'title': _("Changed"),
+                    'field': field_desc,
+                }
+            message.account_audit_log_preview = audit_log_preview
+
+    @api.depends('model', 'res_id')
+    def _compute_account_audit_log_move_id(self):
+        messages_of_account_move = self.filtered(lambda m: m.model == 'account.move' and m.res_id)
+        (self - messages_of_account_move).write({
+            'account_audit_log_move_id': False,
+            'show_audit_log': False,
+        })
+        moves = self.env['account.move'].sudo().search([
+            ('id', 'in', messages_of_account_move.mapped('res_id')),
+            ('company_id.check_account_audit_trail', '=', True),
+        ])
+        moves_by_id = {m.id: m for m in moves}
+        for message in messages_of_account_move:
+            message.account_audit_log_move_id = moves_by_id.get(message.res_id, False)
+            message.show_audit_log = bool(moves_by_id.get(message.res_id, False))
+
+    def _search_account_audit_log_move_id(self, operator, value):
+        is_set = False
+        if operator == '!=' and isinstance(value, bool):
+            is_set = True
+        elif operator not in ['=', 'ilike'] or not isinstance(value, str):
+            raise UserError(_('Operation not supported'))
+        move_domain = []
+        if not is_set:
+            move_domain += [('name', operator, value)]
```

Comment:
This is similar and more readable but it also shows that you didn't care about the value of `value` when the operator was `!=`

```suggestion
        if operator == '!=' and isinstance(value, bool):
            move_domain = []
        elif operator in ['=', 'ilike'] and isinstance(value, str):
            move_domain = [('name', operator, value)]
        else:
            raise UserError(_('Operation not supported'))
```